app/src/main/java/mozilla/org/webmaker/storage/MemStorage.java

```diff
@@ -1,61 +1,20 @@
 package mozilla.org.webmaker.storage;
 
-import android.app.ActivityManager;
-import android.content.Context;
 import android.util.LruCache;
 
 public class MemStorage {
 
-    private static Context context;
-    private static int cacheSize = 5 * 1024 * 2014;
+    private static int cacheSize = 24 * 1024 * 2014;
```

Comment:
what's this new number based on? (we might want to document that in the code so we know what it should reflect and whether it's gone stale)